python_modules/dagster/dagster_tests/execution_tests/pipes_tests/test_subprocess.py

```diff
@@ -207,13 +207,13 @@ def script_fn():
         context.report_asset_materialization(data_version="alpha", asset_key="bar")
 
     @multi_asset(specs=[AssetSpec("foo"), AssetSpec("bar")])
-    def foo_bar(context: AssetExecutionContext, ext: ExtSubprocess):
+    def foo_bar(context: AssetExecutionContext, ext: PipesSubprocess):
```

Comment:
might as well rename the arg from "ext" -> "pipes_subprocess_client" or whatever

Reply:
yeah at this point would rather do that at tip of stack.

I'll keep a list of minor clean up items to do.